src/rockstor/system/shell.py

```diff
@@ -0,0 +1,64 @@
+"""
+Copyright (c) 2012-2016 RockStor, Inc. <http://rockstor.com>
+This file is part of RockStor.
+
+RockStor is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published
+by the Free Software Foundation; either version 2 of the License,
+or (at your option) any later version.
+
+RockStor is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from osi import run_command
+from services import service_status
+import shutil
+from tempfile import mkstemp
+import re
+
+SHELL_CONFIG = '/etc/sysconfig/shellinaboxd'
+SYSTEMCTL = '/usr/bin/systemctl'
+
+def update_shell_config(shelltype='LOGIN', css='white-on-black'):
+
+    fh, npath = mkstemp()
+    with open(npath, 'w') as tfo:
+        #Write shellinaboxd default config
+        tfo.write('# Shell In A Box configured by Rockstor\n\n')
+        tfo.write('USER=root\n')
+        tfo.write('GROUP=root\n')
+        tfo.write('CERTDIR=/var/lib/shellinabox\n')
+        tfo.write('PORT=4200\n')
```

Comment:
Can we read the port number from settings.py instead? If you like to shoot for perfection, you can read all four attributes from it :)